amplitude/types/config.go

```diff
@@ -5,22 +5,25 @@ import (
 )
 
 type Config struct {
-	APIKey             string
-	FlushInterval      time.Duration
-	FlushQueueSize     int
-	FlushMaxRetries    int
-	Logger             Logger
-	MinIDLength        int
-	ExecuteCallback    func(result ExecuteResult)
-	ServerZone         ServerZone
-	UseBatch           bool
-	StorageFactory     func() EventStorage
-	OptOut             bool
-	Plan               *Plan
-	IngestionMetadata  *IngestionMetadata
-	ServerURL          string
-	ConnectionTimeout  time.Duration
-	MaxStorageCapacity int
+	APIKey                 string
+	FlushInterval          time.Duration
+	FlushQueueSize         int
+	FlushSizeDivider       int
+	FlushMaxRetries        int
+	Logger                 Logger
+	MinIDLength            int
+	ExecuteCallback        func(result ExecuteResult)
+	ServerZone             ServerZone
+	UseBatch               bool
+	StorageFactory         func() EventStorage
+	OptOut                 bool
+	Plan                   *Plan
+	IngestionMetadata      *IngestionMetadata
+	ServerURL              string
+	ConnectionTimeout      time.Duration
+	MaxStorageCapacity     int
+	RetryBaseInterval      time.Duration
```

Comment:
Are these also used in other SDKs?

Reply:
For example, in Amplitude-Python some values are configured, some are constants:

https://github.com/amplitude/Amplitude-Python/blob/main/src/amplitude/config.py#L124
```
    @property
    def server_url(self):
        if self._url:
            return self._url
        return constants.SERVER_URL[self.server_zone][constants.BATCH if self.use_batch else constants.HTTP_V2]

    @server_url.setter
    def server_url(self, url: str):
        self._url = url
```

https://github.com/amplitude/Amplitude-Python/blob/main/src/amplitude/constants.py#L50
```
CONNECTION_TIMEOUT = 10.0  # seconds float
MAX_BUFFER_CAPACITY = 20000
```

https://github.com/amplitude/Amplitude-Python/blob/main/src/amplitude/storage.py#L118
```
    def _get_retry_delay(self, retry: int) -> int:
        if retry > self.configuration.flush_max_retries:
            return 3200
        if retry <= 0:
            return 0
        return 100 * (2 ** ((retry - 1) // 2))
```